src/server.c

```diff
@@ -5929,6 +5931,8 @@ sds genRedisInfoString(dict *section_dict, int all_sections, int everything) {
             "total_writes_processed:%lld\r\n"
             "io_threaded_reads_processed:%lld\r\n"
             "io_threaded_writes_processed:%lld\r\n"
+            "client_query_buffer_limit_reached:%lld\r\n"
+            "client_output_buffer_limit_reached:%lld\r\n"
```

Comment:
maybe we can find a clearer name?
i.e. one that suggests it's a counter, and maybe indicates that these are disconnections?
how about "query_buffer_limit_client_disconnections"?
hoping someone has a better suggestion...

Reply:
since the config name is `client-query-buffer-limit`, how about `client_query_buffer_limit_disconnections` or just `client_query_buffer_limits`?

Reply:
out of these two, i'd prefer `client_query_buffer_limit_disconnections`, i wish the word "disconnections" wasn't that long :smile: , maybe someone else has a suggestion, if not, let's take that one.

Reply:
I think part of it is we are trying to make it grammatically correct, which isn't really a standard in info. We could do something like:
`clients_disconnected_query_buffer_limit_reached`. From a wording perspective, I find it easier to follow that it's a counter of clients disconnected by the server when the words are together, then subtype it with a reason. We do this fragmentation very often, `used_memory ... _scripts`.

Reply:
and isn't `clients_disconnected_query_buffer_limit` enough?

Reply:
> and isn't clients_disconnected_query_buffer_limit enough?

Sure, I personally want it to be "query_buffer_limit_reached" or "exceeded", but don't feel that strongly.

Reply:
@yossigo please suggest / advise / approve

Reply:
Also prefer `client_query_buffer_limit_disconnections`.